carma-messenger-core/emergency_response_vehicle_plugin/test/test_emergency_response_vehicle_plugin.cpp

```diff
@@ -207,11 +207,11 @@ namespace emergency_response_vehicle_plugin{
 
         worker_node->handle_on_shutdown();
     }
-
+    /**
```

Comment:
I know this might require a bit of repetitive work on the related carma-platform PR, but would it be worth adding a comment to each commented-out test that explains why this unit test has been commented out for when future developers come across it? 

Something similar to:
```
// This unit tests has been temporarily disabled to support Continuous Improvement (CI) processes.
// Related GitHub Issue: <link>
```